src/network/service.rs

```diff
@@ -372,7 +381,7 @@ where
                     .await;
 
                 let peer_id = self.libp2p.connection_peer_id(*id).await;
-                let chain_index = overlay_network_index / 2;
+                let chain_index = overlay_network_index / 3;
```

Comment:
Nitpick: isn't it better to have this as a constant defined in this file and used in all of those places?

Reply:
Indeed, I'll do this on top of this.